src/widget/whighlightinggroup.h

```diff
@@ -0,0 +1,59 @@
+#ifndef WHIGHLIGHTINGGROUP_H
+#define WHIGHLIGHTINGGROUP_H
+
+#include "widget/wwidgetgroup.h"
+
+class SkinContext;
+
+// The WHighlightingGroup can be used to highlight a group of widgets by
+// changing the background or any other css style option.
+// Example skin:
+// <HighlightingGroup>
+//  <Id>1</Id>
+//  <BackPathHighlighted>style/style_bg_effect1_high</BackPathHighlighted>
+//  <Connection>
+//   <ConfigKey>[EffectRack1_EffectUnit1],single_effect_focus</ConfigKey>
+//   <BindProperty>highlightedId</BindProperty>
+//  </Connection>
+// </HighlightingGroup>
+
+class WHighlightingGroup : public WWidgetGroup {
+    Q_OBJECT
+  public:
+    explicit WHighlightingGroup(QWidget* pParent = nullptr);
+
+    // The highlight property is used to restyle the widget with CSS.
+    // The declaration #MyGroup[highlight="1"] { } will define the style
+    // for the highlighted state. Note: The background property does not
+    // support color schemes for images, a workaround is to set the background
+    // image via <BackPath> and <BackPathHighlighted> from the skin.
+    Q_PROPERTY(int highlight READ getHighlight WRITE setHighlight NOTIFY highlightChanged)
+
+    // The highlightedId property is used to select the highlighted widget
+    // group by its Id, which can be set from skin by <Id>4</Id>
+    Q_PROPERTY(int highlightedId READ getHighlight WRITE setHighlightedId NOTIFY highlightChanged)
```

Comment:
Is there a confusion due to the ID member? Maybe the name is wrong. It is the common ID of all widgets that are highlighted if the connected CO has the ID value. 

If you have two HighligtingGroups with ID=4 both are highlighted if the connected control has the value 4.